openmetadata-ui/src/main/resources/ui/src/pages/TasksPage/shared/Assignees.tsx

```diff
@@ -36,13 +42,21 @@ const Assignees: FC<Props> = ({
   onChange,
   options,
   disabled,
+  isSingleSelect = false,
```

Comment:
What's `isSingleSelect`?

Reply:
there can be a multi-select option for assignees or we can have single select.